### PR TITLE
feat(playground): add buttons to select a backend

### PR DESCRIPTION
**Pull request by CharlesRngrd**
Hi Toucan,

Here is my PR to add buttons to select a backend.
When Pandas is selected, we remove the mongo query.

test

### CODE REVIEW

File: playground/dist/index.html [hunk 1]
```diff
@@ -105,21 +181,37 @@
...
         <div class="debug-panel__code">
           <div class="debug-panel__code-pipeline">
             <div>Pipeline</div>
             <pre>{{ pipelineAsJSON }}</pre>
           </div>
           <div class="debug-panel__code-translated-pipeline">
             <div>Translated pipeline</div>
-            <pre>{{ mongoQueryAsJSON }}</pre>
+            <pre v-if="isSelectedTranslator('mongo50')">{{ mongoQueryAsJSON }}</pre>
+            <pre v-else>Query executed but translation not configured for this backend</pre>
```
Comment (davinov) [inline playground/dist/index.html:214]: Indeed, translation makes no sense for pandas "executor".
I suggest you to hide completely translation panel: `<div class="debug-panel__code-translated-pipeline" v-if="!isSelectedTranslator('pandas')">`

File: playground/dist/index.html [hunk 2]
```diff
@@ -74,14 +68,96 @@
...
+        transition: top .5s ease-out;
       }
 
       .debug-panel__code {
         display: flex;
         background-color: white;
+        height: inherit;
+        padding: 10px;
+        height: calc(100% - 56px);
+        overflow-y: scroll;
```
Comment (davinov) [inline playground/dist/index.html:160]: Nitpick: Is `auto` not sufficient?

**Review by davinov — CHANGES_REQUESTED** [review]
Nice! Can't wait to add the SQL translation to this panel too :tada: 
Thanks for this contrib :clap: Here are just 2 small details to make it perfect :sparkles:

**Comment by CharlesRngrd** [comment]
- Removing the translation panel seems a little bit weird for me  -> @davinov Is it ok for you ?
- `height: auto` doesn't worked for me -> @davinov Can I keep my original code ?

test

**Review by davinov — APPROVED** [review]
OK! Let's keep it like this for now.
Thanks again for this improvement :)

**Comment by CharlesRngrd** [comment]
> OK! Let's keep it like this for now. Thanks again for this improvement :)

Cool ;)
@davinov I fixed the conflicts, now you can merge.

**Comment by CharlesRngrd** [comment]
@davinov I updated the CSS because right now the pagination is hidden by the bottom toolbar...

**Pull request merged**